lib/stringtheory/src/lib.rs

```diff
@@ -206,23 +188,12 @@ impl From<&str> for MetaString {
                 inner: Inner::Inlined(i),
             },
             None => Self {
-                inner: Inner::Shared(bytes::Bytes::copy_from_slice(s.as_bytes())),
+                inner: Inner::Owned(s.to_owned()),
```

Comment:
These are functionally equivalent in that they both allocate the same exact amount.. although this change does mean overall that things are less efficient when we clone `MetaString` since there's no variant that can be shared cheaply like `Bytes` can be.